### ISSUE TITLE
Install in other folder instead of 'public' folder

### ISSUE THREAD

**Issue by chinhnm** [mbo2olivier/mukadi-wordpress-bundle#5]
Hi mbo2oliver, First thank for awsome bundle. I have tried to install in other folder (I want to install it in my custom folder). Just tried change the json file, remove the file to the custom folder but it still run in public/index.php and return a lot of bugs.

**Comment by mbo2olivier** [comment]
Hello @chinhnm and thank for using this bundle.
i don't understand properly: what are you trying to install in a custom folder ? the wordpress files ? if so, you must first set the "wordpress-install-dir" in your composer.json and then install the bundle, after that you have to change the WP_DIR in the .env file.

**Comment by mbo2olivier** [comment]
Sorry I had not understood the question well, if you want to Install in other folder instead of 'public' folder, you must fix it in the bundle configuration file at config/package/mukadi_wordpress.yaml too.

**Comment by chinhnm** [comment]
Hi @mbo2olivier, Of course, after ask this question, I have changed the set up ( "wordpress-install-dir" in my composer.json, config/package/mukadi_wordpress.yaml too ...) but the code still run through 'public/index.php' and have conflict bug in here. Why I have to install in custom folder? Yes, I have set up Webpack for css,js after run command the code will render in here and my customer want me to install it in 'blog' folder instead of 'public folder'.
